content/articles/Introduction-to-Containers-and-Docker/index.md

```diff
@@ -0,0 +1,87 @@
+To comprehend what `Docker` is and why it is used, it is necessary to first understand what `containers` are and what problems they solve. Containers are completely isolated environments built on top of an existing operating system to provide a `virtual barrier` between the app running inside and the rest of the world. Docker, on the other hand, is a piece of software that allows us to manage the lifetime of these containers.
+
+### Table of content
```

Comment:
Any prerequisites needed for one to understand the article? Consider your reader being someone who is reading about Docker for the first time.